torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -2255,11 +2261,13 @@ def generate_extern_kernel_alloc_and_find_schema_if_needed_jit(
             self.load_custom_op_wrapper()
 
             assert output_args is not None, "output_args should not be None"
-            num_args = len(raw_args)
+            # raw arg may be None
```

Comment:
Do we still need to pass the raw arg if it is None?

Reply:
> Do we still need to pass the raw arg if it is None?

No, we don't. The change above (line 2217 and 2218) skips any None raw arg.

Reply:
If we have raw args like `buf0, a=None, b=1`, I think we should still pass in the second parameter as None/Py_None?